jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/redis/RedisConfiguration.java

```diff
@@ -100,5 +102,10 @@ public CacheManager cacheManager(JedisConnectionFactory jedisConnectionFactory,
 	   return RedisCacheManager.RedisCacheManagerBuilder.fromConnectionFactory(jedisConnectionFactory)
 	                    .cacheDefaults(redisCacheConfiguration).build();
     }
-	
+
+    @Bean
+	@ConditionalOnExpression("'${bcgov.access.input.plugin}' == 'sftp'")
```

Comment:
add also `@ConditionalOnMissingBean(ConcurentMetaDataStore.class)` annotation, so you can let the client of the lib override the config ;-)

Reply:
yes. thanks. I learn new things.
Changed.